distribute/makefile

```diff
@@ -0,0 +1,32 @@
+PYTHON=.venv/bin/python
+PIP=.venv/bin/pip
+
+.venv:
+	python -m venv .venv
+	${PIP} install \
+		-r requirements.txt \
+		-r distributor/dataplatform/requirements.txt
+
+.env:
+	@echo "SOURCE_NAME=# Name of the API gateway owner" >> .env
+	@echo "SOURCE_IDENTIFIER=#URL to the API gateway owner containing json information about them" >> .env
+	@echo "# Dataplatform distributor" >> .env
+	@echo "ORIGO_ENVIRONMENT=dev" >> .env
+	@echo "ORIGO_USERNAME=# Byrbruker" >> .env
+	@echo "ORIGO_PASSWORD=# Byrpassord" >> .env
+	@echo "Remember to run 'export $$(cat .env)'"
+
+init: .venv .env
+
+../data/:
+	mkdir -p ../data
+
+latest.ttl: ../data/
+	${PYTHON} ../tools/json_merger.py ../data/ | ${PYTHON} ../tools/latest_provider/scripts/turtle.py > latest.ttl
```

Comment:
Jeg får
```bash
$ make latest.ttl 
.venv/bin/python ../tools/json_merger.py ../data/ | .venv/bin/python ../tools/latest_provider/scripts/turtle.py > latest.ttl
Traceback (most recent call last):
  File "../tools/latest_provider/scripts/turtle.py", line 4, in <module>
    from rdflib import URIRef, Graph, Literal, RDF, BNode
ModuleNotFoundError: No module named 'rdflib'
make: *** [makefile:25: latest.ttl] Error 1
```

Først trodde jeg det kanskje var min vanlige python bug, men jeg ser at turtle.py bruker bruker rdflib uten at det er spesifisert i en requirements.txt som er involvert fra denne koden. Med andre ord: distribute/.venv/lib/python3.8/site-packages inneholder ikke rdf-lib, noe den må for at turtle.py skal funke i dette venv-et.

For at det skal funke må enten
* requirements.txt i distributor-mappe inneholde rdflib, eller
* requirements.txt i mappa til turtle.py inneholde rdflib, som kanskje kan være mer logisk, men da må man kjøre pip install -r på den mappa også.

Evt du kan lage et bibliotek.

Har ikke en god forståelse over prosjektet med disse PR-endringene atm, så vet ikke hva som er best.

Hvis alt dette stemmer, så tipper jeg at det funker for deg fordi du har rdflib installert med --user eller sudo. I så fall ville jeg vurdert å avinstallere det, slik at du oppdager slike ting. Men ook, her det masse antakelser.

Reply:
Godt poeng. Den var der fordi jeg ville ha en snarvei for å generere en turtle fil, men den hørte ikke hjemme der altså. Fjernet latest.ttl targeten fra den makefilen